packages/plugin-packages/mashroom-portal-sandbox-app/src/js/components/MessageBusSendForm.tsx

```diff
@@ -26,6 +26,8 @@ type Props = {
 
 export default class MessageBusSendForm extends PureComponent<Props> {
 
+    sbAutoTest = Number(new URLSearchParams(window.location.search).get('sbAutoTest'));
```

Comment:
please don't directly access the location but use the portalStateService - also there is already a function getQueryParams() in js/utils